platform-workflow-module/src/test/java/com/softicar/platform/workflow/module/workflow/task/WorkflowUserFieldTest.java

```diff
@@ -0,0 +1,111 @@
+package com.softicar.platform.workflow.module.workflow.task;
+
+import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.workflow.module.demo.AGWorkflowDemoObject;
+import com.softicar.platform.workflow.module.test.AbstractTestObjectWorkflowTest;
+import com.softicar.platform.workflow.module.workflow.item.AGWorkflowItem;
+import com.softicar.platform.workflow.module.workflow.task.delegation.AGWorkflowTaskDelegation;
+import com.softicar.platform.workflow.module.workflow.task.delegation.AGWorkflowTaskDelegationLog;
+import com.softicar.platform.workflow.module.workflow.transients.field.WorkflowUserField;
+import java.util.Set;
+import org.junit.Test;
+
+public class WorkflowUserFieldTest extends AbstractTestObjectWorkflowTest {
+
+	private final AGWorkflowTask task;
+	private final AGWorkflowItem item;
+	private final AGUser user2;
+	private final AGWorkflowDemoObject object;
+	private final int TEST_AGE = 3;
```

Comment:
I think we should add a `_DAYS` suffix here.

Reply:
Done

Reply:
Does not seem to be the case. Is there still something to be pushed?

Reply:
nvm